examples/permissioned-token/assets/permissions_new.py

```diff
@@ -0,0 +1,17 @@
+# add new permissions contract logic here (eg. KYC)
+import sys
+sys.path.insert(0,'..')
+
+from algobpy.parse import parse_params
+from pyteal import *
+
+def approval_program():
+    program =  Return(Int(1))
+    return program
+
+if __name__ == "__main__":
+    # Overwrite params if sys.argv[1] is passed
+    if(len(sys.argv) > 1):
+        params = parse_params(sys.argv[1], params)
+
+    print(compileTeal(approval_program(), Mode.Application, version=3))
```

Comment:
This is super easy program, we could just use TEAL instead of PyTEAL

Reply:
This was meant as a placeholder program to add new permissions code. I think pyTEAL looks much better to eye than regular teal.

Anyway, updating